_python_workout_code/7.1.py

```diff
@@ -0,0 +1,5 @@
+def  join_numbers(count):
+    list = [i for i in range(count)]
+    return ','.join(map(str, list))
```

Comment:
even though I don't think you've needed the transformation to list below, you are doing the second loop here.
to avoid this, you could have written a one-liner `return ', '.join(str(x) for x in range(5))`